frontends/mit-open/src/pages/HomePage/BrowseTopicsSection.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react"
+import { Container, styled, theme, Button, Typography } from "ol-components"
+import { Link } from "react-router-dom"
+import { useTopicsList } from "api/hooks/topics"
+import {
+  RiPaletteLine,
+  RiSeedlingLine,
+  RiBriefcaseLine,
+  RiMacbookLine,
+  RiBarChartBoxLine,
+  RiUserSearchLine,
+  RiArrowRightSLine,
+} from "@remixicon/react"
+
+const ICONS = [
+  RiPaletteLine,
+  RiSeedlingLine,
+  RiBriefcaseLine,
+  RiMacbookLine,
+  RiBarChartBoxLine,
+  RiUserSearchLine,
+  RiArrowRightSLine,
+]
+
+const Section = styled.section`
+  background:
+    linear-gradient(270deg, rgba(243 244 248 / 0%) 0%, rgb(243 244 248) 100%),
+    url("/static/images/open-bg-texture-wgradient.jpg") lightgray 50% / cover
+      no-repeat;
+  background-position: center right;
+  padding: 80px 0;
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    padding: 40px 0;
+  }
+`
+
+const Title = styled(Typography)`
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    text-align: center;
+  }
+`
+
+const Topics = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px 24px;
+  margin: 40px 0;
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    gap: 5px;
+    margin: 24px 0;
+  }
+`
+
+const TopicBox = styled(Link)`
+  flex: 0 1 calc(100% * (1 / 3) - 16px);
+  padding: 28px 30px;
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    flex: 0 1 100%;
+    padding: 18px 15px;
+  }
+
+  border-radius: 5px;
+  border: 1px solid ${theme.custom.colors.lightGray2};
+  background: ${theme.custom.colors.white};
+  overflow: hidden;
+`
+
+const TopicBoxContent = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 10px;
+
+  svg {
+    flex: 0 0 20px;
+  }
+
+  font-weight: 500;
+  font-size: ${theme.typography.pxToRem(16)};
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    font-size: ${theme.typography.pxToRem(14)};
```

Comment:
Are we planning that the typography element settings will have breakpoints built in? The designs have different font size and same line heights, which we don't have a match for.

Reply:
Please use `subtitle1` for desktop and `subtitle2` for mobile. subtitle1 matches the desktop, subtitle2 just has a 2px smaller line height. I've asked @steven-hatch / @mbilalmughal to update the designs. 

Apologies these weren't named styles in the figma. If that happens again, please (1) use the closest named style **and** (2) reach out to @steven-hatch / @mbilalmughal to let them know the figma needs to be updated.

In general, we want to use the named styles for consistency but also for because the CSS font weights in figma are sometimes wrong. (Not always, but sometimes). See https://github.com/mitodl/mit-open/pull/889#discussion_r1598820897 for details.

-----

### Responsive fonts
As to your original question of responsive fonts.. (e.g., `subtitle1` automatically gets smaller for certain screen sizes). IMO, that would be great, but for now we don't. 

 @steven-hatch / @mbilalmughal If we do have rules like "subtitle1 **always** becomes subtitle2 on mobile screens" that would be helpful to know.

Reply:
Sounds good, have changed.